index.js

```diff
@@ -63,7 +63,7 @@ PublishRelease.prototype.publish = function publish () {
           'User-Agent': 'publish-release ' + pkg.version + ' (https://github.com/remixz/publish-release)'
         }
       }, function (err, res, body) {
-        if (err) return callback(err)
+        if (err || body.errors) return callback(err || body.errors)
```

Comment:
I think is better pass ahead the whole error object. As you can see in this [message](https://github.com/remixz/publish-release/pull/10#issue-118297225) the `erros` property does not have the whole information about the error.